packages/create-cloudflare/src/frameworks/next/index.ts

```diff
@@ -72,16 +73,65 @@ const configure = async (ctx: PagesGeneratorContext) => {
 	writeFile(handlerPath, handlerFile);
 	updateStatus("Created an example API route handler");
 
+	const installEslintPlugin = await shouldInstallNextOnPagesEslintPlugin(ctx);
+
+	if (installEslintPlugin) {
+		await writeEslintrc(ctx);
+		updateStatus("eslint-plugin-next-on-pages added to eslintrc");
+	}
+
 	// Add some dev dependencies
 	process.chdir(projectName);
-	const packages = ["@cloudflare/next-on-pages@1", "vercel"];
+	const packages = [
+		"@cloudflare/next-on-pages@1",
+		"vercel",
+		...(installEslintPlugin ? ["eslint-plugin-next-on-pages"] : []),
+	];
 	await installPackages(packages, {
 		dev: true,
 		startText: "Adding the Cloudflare Pages adapter",
 		doneText: `${brandColor(`installed`)} ${dim(packages.join(", "))}`,
 	});
 };
 
+// Note: this isn't a generic helper since it's Next specific, this can be
+//       made into a generic helper by checking all possible types of configs
+//       (including a field in the package.json)
+//       (and potentially returning what type of config is used)
+export const usesEslint = (projectRoot = ".") => {
+	return existsSync(`${projectRoot}/.eslintrc.json`);
+};
+
+export const shouldInstallNextOnPagesEslintPlugin = async (
+	ctx: PagesGeneratorContext
+): Promise<boolean> => {
+	if (!usesEslint(ctx.project.name)) return false;
+
+	return await processArgument(ctx.args, "eslint-plugin" as keyof C3Args, {
+		type: "confirm",
+		question: "Do you want to use the next-on-pages eslint-plugin?",
+		label: "",
```

Comment:
is the empty label ok?

Reply:
Nope, it's not :D 

Without this, there will be not status label shown when this operation is complete. I'd suggest `plugin` or `eslint-plugin`.

Reply:
ahhhh ok I see 🤔 

Ok this is how I have it now:
![Screenshot 2023-07-14 at 21 44 14](https://github.com/cloudflare/workers-sdk/assets/61631103/71477243-5dff-4df6-aba2-eab01641b9c4)

And this is how it should be?
![Screenshot 2023-07-14 at 21 46 00](https://github.com/cloudflare/workers-sdk/assets/61631103/7c4488f8-ee0d-459b-a1bb-df27ff27af2e)

What about the `eslint-plugin-next-on-pages added to eslintrc` message? is that ok? or should it be avoided?

Reply:
label added and redundant message removed 🙂👍